### DISCUSSION TITLE
Bus stop code or number quest never shown in UK

### DISCUSSION THREAD

**Discussion by boothym** [streetcomplete/StreetComplete#4545] [General]
The quest "What number or code identifies this stop?" is never show in the UK - just wondering what the reason for this is?

Because UK bus stops do have a reference code, so not sure why this wouldn't be asked?

**Comment by mnalis** [comment]
It is disabled in all countries **_except_** those for which it has been confirmed it makes sense:

https://github.com/streetcomplete/StreetComplete/blob/00f00f60b192bda332dae5249b5fdcd099148f29/app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_ref/AddBusStopRef.kt#L20-L27

Because it often varies a lot by specific part of the country (i.e. something that is common in England might be non-existant in Northern Ireland, or vice versa), it would make sense to get a broader opinion on UK community OSM communication channels. If the consensus is it indeed makes sense to enable the quest in `GB`, a new [issue](https://github.com/streetcomplete/StreetComplete/issues) could be opened summarizing and linking to that discussion.
